mobile/README.md

```diff
@@ -0,0 +1,15 @@
+# GnoSocial Mobile APP
+
+## Requirements
+
+- Expo Requiments: https://docs.expo.dev/get-started/installation/
+
+## 🚀 How to use
+
+```sh
+# to run on ios:
+yarn ios 
+
```

Comment:
For iOS, is it also necessary to do `pod install` ? If yes, then should put this in the README.